definitions/features/pulpcore.rb

```diff
@@ -50,16 +37,4 @@ def self.pulpcore_common_services
     ]
     common_services + pulpcore_resource_manager_service
```

Comment:
you can't delete `pulpcore_resource_manager_service` below, but still reference it here ;-)